main/main.go

```diff
@@ -134,11 +129,6 @@ func main() {
 	// We want to remove logr.Logger first step to move to logrus
 	// in the future we will have only one of them
 	newLogger := logrus.New()
-	logger.SetFormatter(&logrus.TextFormatter{
```

Comment:
This is meant to be done to `newLogger`?

Reply:
no I originally added this for the `logger`. After refactoring the loggers, `logger` became unused and we forgot to remove the `logger` and this.

Reply:
Ok. I was just wondering about the format. Looks weird with two different formats mixed together. Anyway to converge on one? Not for this PR though.